examples/example_vms.py

```diff
@@ -22,35 +23,38 @@ def main():
     # List existing VMs
     pprint(api.list_namespaced_virtual_machine(NAMESPACE))
 
+    # List existing VMIs
+    pprint(api.list_namespaced_virtual_machine_instance(NAMESPACE))
+
     # Create new one
-    vm = el.read_yaml_file("vm-ephemeral.yaml")
-    pprint(api.create_namespaced_virtual_machine(vm, NAMESPACE))
+    vmi = el.read_yaml_file("vmi-ephemeral.yaml")
```

Comment:
Is it legit to create vmi before a vm was create for it ?

Reply:
Yes, it is OK